test/main.js

```diff
@@ -1,13 +1,16 @@
-
+const program = require('commander');
 const TestManager = require('./lib/testManager');
 
-if (process.argv.indexOf('-L') > -1) {
-  const
-    language = process.argv[process.argv.indexOf('-L') + 1],
-    testManager = new TestManager(language);
+program
+  .version(require('../package.json').version)
+  .option('-L, --language <language>', '[Mandatory] The language to test. Available languages are js, go')
+  .option('-O, --only <test_path>', 'The path of a single test to execute, relative to src/sdk-reference')
```

Comment:
uppercase `O` vs lowercase `o` in the README

Reply:
yep, this is actually right: the README explains the options of the `run_test.sh`, while here you're looking at the `test/main.js` program. I wanted to harmonize them anyways, but I forgot! 😓